docs/release-notes/mia-platform-v13.mdx

```diff
@@ -7,3 +7,140 @@ image: "img/release-note-link-preview.png"
+
+To learn more, please refer to the [documentation]().
+
+INSERT THE WORKING LINK
+
+### React to Console actions using Webhooks
+
+The variety of Console events suitable for webhook configuration constantly increases, allowing you to react and build custom flows and logic each time the event occurs. 
+
+With the introduction of user management events and the introduction of the history of each webhook to facilitate monitoring and debugging, users can now use webhooks to extend Console functionality.
+
+INSERT AN IMAGE -> mettiamo la history dei webhook!
+
+To learn more, please refer to the [documentation](/development_suite/webhooks-and-events/webhooks.md#event-history).
+
+### Connect Backstage portal to Mia-Platform
+
+On Backstage Marketplace you will find the Mia-Platform plugin for integrating Mia-Platform components into Backstage components.  
+It keeps your information synchronized effortlessly and gains insights into your Company's domains, Projects, deployment environments, and repository links, all while exploring their interrelations.
+
+## Simplify Composability
+
+Mia-Platform Suite empowers development teams to enhance software composability by facilitating streamlined reuse and embracing scalable approaches. With Mia-Platform v13, software composability is even more NoCode! 
+The main benefits for the user can be summarized by the following:
+
+- Democratizing access to the IDP's internal tools and utilities
+- Providing user-friendly interfaces to facilitate configuration, management and monitoring of your composable software
+- Enriching the management of IDP with even more functionalities
+
+Let's go in deep with the new Features:
+
+### Enrich Meta Data and list all your data assets: Data Catalog
+
+Let's welcome our first release of **Data Catalog**, designed to provide the data governance team and data analysts with a centralized location where all metadata concerning the data assets of an organization are collected.
+
+ADD IMAGE -> dove facciamo vedere lo schemino architetturale
+
+Set up our Data Catalog Agent, to connect to different external sources, such as Oracle Database, Salesforce, Postgres Database, MySQL Database, Microsoft SQL Server Database, and to extract all metadata existing on your organization's System of Records. 
+
+Through Mia-Platform Console, it is possible to:
+
+- Keep metadata up to date with the external System of Records
+- Navigate data models in a centralized place
+- Enrich metadata with descriptions and GDPR regulation properties for each table field
+- Export your data models in .csv format
+- Make metadata usable via Data Catalog API from third-party systems and custom frontends
+
+To learn more, please refer to the [documentation](link).
+
+### Seamless Runtime Management with Fast Data Control Plane
+
+Mia-Platform Fast Data has been empowered with a Runtime Management solution, **Fast Data Control Plane**.  
+
+ADD IMAGE
```

Comment:
We made a GIF 
![Gif fast data con border radius](https://github.com/mia-platform/documentation/assets/84580717/66bbc8f8-9564-4d02-b4ca-bf6ed25d77af)

Reply:
@albertotessarotto we can use this gif!